rnd/autogpt_builder/yarn.lock

```diff
@@ -4,31 +4,31 @@
 
 "@alloc/quick-lru@^5.2.0":
   version "5.2.0"
-  resolved "https://registry.yarnpkg.com/@alloc/quick-lru/-/quick-lru-5.2.0.tgz#7bf68b20c0a350f936915fcae06f58e32007ce30"
+  resolved "https://registry.npmjs.org/@alloc/quick-lru/-/quick-lru-5.2.0.tgz"
```

Comment:
I'm very curious what caused all of these changes, maybe a different version of yarn?

Reply:
fixed